packages/react/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -115,7 +117,7 @@ interface DropdownListProps {
     label?: string;
     name?: string;
     /**
-     * { disabled?: boolean, value: string; label: string; }[]
+     * { value: string; label: string; caption?: string; disabled?: boolean }[]
```

Comment:
Je ne comprend pas pourquoi on a ca? Il me semble que les IDE integre l'intellisense TypeScript meme si le projet n'est pas en TS.

Reply:
C'est juste pour mieux documenter dans le storybook... Notre documentation est un peu inégale, je crois qu'il faudrait avoir une bonne ligne d'explication sur chacune des props.

Pour ce qui est des interfaces plus complexes, je n'ai pas vraiment de préférence. J'avais l'impression que ça donnait un coup de main d'avoir le détail sans aller voir le code...

Reply:
Perso, ca me semble pas une bonne idee de dupliquer les definitions des interfaces dans les comment block de meme, ca me semble unfit, le block sert plus a expliquer l'usage en general du prop.  J'ai plus l'impression que c'est un probleme de Storybook qui manque quelque chose, ou faudrait un genre de https://typedoc.org/example

Reply:
T'as raison, la duplication c'est problématique... J'ai changé cette interface-là 2-3 fois en développant le component et j'ai vraiment pas tout le temps pensé à aller mettre à jour la doc!

Je vais essayer d'explorer ce qui serait possible avec Storybook, y'a p-e une config ou un plugin qui nous permettrait d'y arriver. I'll report back!

Reply:
J'ai rien trouvé pour l'instant mais j'ai quand même retiré ça, ton point est valide!